bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/DiscoveryResultImpl.java

```diff
@@ -226,4 +239,14 @@ public long getTimestamp() {
     public long getTimeToLive() {
         return timeToLive;
     }
+
+    @Override
+    public void discoveredBy(String discoveryService) {
```

Comment:
`discoveryService` is probably a bad name here - maybe call it discoverer, fqcn or classname instead?